utils/wasm-builder/src/wasm_project.rs

```diff
@@ -506,22 +390,14 @@ extern "C" fn metahash() {{
         // Tuple with PathBuf last wasm & opt.wasm
         let mut wasm_paths: Option<(PathBuf, PathBuf)> = None;
```

Comment:
no need in this tuple now?

Reply:
why not? the function still returns tuple with paths to the original wasm and optimized ones